src/Features/Core/Portable/UseExplicitTupleName/UseExplicitTupleNameDiagnosticAnalyzer.cs

```diff
@@ -49,7 +49,7 @@ private void AnalyzeOperation(OperationAnalysisContext context)
             var field = fieldReferenceOperation.Field;
             if (field.ContainingType.IsTupleType)
             {
-                if (field.CorrespondingTupleField.Equals(field))
+                if (option.Value && field.CorrespondingTupleField.Equals(field))
```

Comment:
it would be better to check this right after getting the option.  with a comment explaining things.

Reply:
Thanks, fixed!